### PR TITLE
Release notes for Service Mesh 1.0.4 release

### PR DESCRIPTION
**Pull request by JStickler**
This PR includes the following changes for the 1.0.4 release:
Moves the Release Notes to first position in the doc set (this will make the Release Notes easier to find on the Customer Portal, where the Service Mesh docs are a single "manual")
Removes Tech Preview content from New Features file
Updates Known issues
Updates Fixed issues
Updates the version in the attributes file

### CODE REVIEW

**Review by brian-avery — APPROVED** [review]

**Comment by yxun** [comment]
We got an OCP 4.3 UI change update in https://issues.redhat.com/browse/MAISTRA-1041 . I will push a commit to add that related change in the PR

**Comment by yxun** [comment]
@JStickler   There is a new OCP 4.3.0 UI step for installation SMCP and SMMR when login as a normal user. Details are in https://github.com/JStickler/openshift-docs/pull/1 . Could you help review and merge that PR into this PR ?

**Comment by JStickler** [comment]
@yxun Let's deal with that update in a separate PR so it doesn't hold up the Release Notes.

**Comment by yxun** [comment]
@JStickler Ok, yes, a separate PR is better. I can also help explain anything unclear in a separate PR later.

File: modules/ossm-rn-fixed-issues.adoc [hunk 1]
```diff
@@ -16,39 +16,38 @@ Result - How has the behavior changed as a result?  Try to avoid “It is fixed
...
+
+* link:https://issues.jboss.org/browse/MAISTRA-348[MAISTRA-348] OpenShift 4 Beta on AWS does not support ingress gateway traffic on ports other than 80 or 443.  If you configure your ingress gateway to handle TCP traffic with a port number other than 80 or 443, you have to use the service hostname provided by the AWS load balancer rather than the OpenShift router as a workaround.
+
+[id="ossm-rn-fixed-issues-kiali_{context}"]
+== Kiali fixed issues
+
+* link:https://issues.jboss.org/browse/KIALI-3096[KIALI-3096] Runtime metrics fail in {ProductShortName}. There is an oauth filter between the {ProductShortname} and Prometheus, requiring a bearer token to be passed to Prometheus before access will be granted. Kiali has been updated to use this token when communicating to the Prometheus server, but the application metrics are currently failing with 403 errors.
```
Comment (neal-timpe) [inline modules/ossm-rn-fixed-issues.adoc]: OAuth is styled with OA capitals earlier. I think it probably should be here too. Not a blocker though.

**Comment by neal-timpe** [comment]
LGTM other than the quick style point.

**Pull request merged**